crates/starknet-server/src/api/http/models.rs

```diff
@@ -63,8 +61,8 @@ pub(crate) struct Time {
 }
 
 #[derive(Serialize)]
-pub(crate) struct PredeployedAccount {
-    pub(crate) initial_balance: u128,
```

Comment:
In the documentation of the pythonic devnet the initial_balance is in scientific notation

Reply:
EDIT: Discussed to make it use BigInt and serialize not in scientific notation. To prevent losing precision

Reply:
I implemented the decimal string approach using bigint: https://github.com/0xSpaceShard/starknet-devnet-rs/pull/30/commits/5a756b5907c35d68270e72ad1cda795c967420ee

I introduced a new trait called `ToDecimalString` instead of using the built-in `ToString` since we already have some stringifying methods. Let me know if I should change this.